contents/founders/what-recruiters-see.md

```diff
@@ -0,0 +1,77 @@
+---
+date: 2024-01-23
+title: What startup recruiters actually see when you apply for a job
+author:
+  - charles-cook
+featuredImage: ../images/blog/transparency.jpeg
+featuredImageType: full
+tags:
+  - Founders
+  - Culture
+  - People  
+---
+
+Last week we opened a new marketing role at PostHog, and we got 300 applicants in the first 2 days. This was high, but not unusual for a startup - my record is 900 in a day, but I bet folks working at giant companies like Google see way way more. Of these 300, 12 got an interview. 
+
+As I sifted through these, I realized a very obvious thing - there is a _much_ bigger gap between what applicants and recruiter experience than either group realizes:
+
+- What applicants and recruiters see on their screens are _completely_ different. This leads to some needlessly funky looking applications - luckily this is easy to solve. 
+- You may spend 30min+ researching a company, crafting a cover letter and applying. At these numbers, I was able to spend **less than 1min** reviewing each application.  And this is just an average - some applications took me 5s(!) to realize they were a no, so I could spend 5min+ reading good ones carefully.
+
+Remember, actual professional recruiters don't just spend their time reviewing applications - they are conducting and scheduling interviews, collating feedback, negotiating offers, onboarding new hires, creating headcount plans, putting up job ads, etc. While it may seem extremely harsh that your application only gets a quick review, bear in mind that we're trying to figure out whether to invest dozens of people hours putting you through the full hiring process. 
+
+By sharing a bit of insight into how the _application_ bit of the hiring process actually works at a startup like PostHog, hopefully I can help increase your chances of landing a first round interview at least! Beyond that, no promises...
+
+## But first, some context
+
+I'm talking about startups with a reasonable degree of brand recognition (ie. not a startup founded last week). In PostHog's case, we are:
+
+- 42 people, $27m raised
+- Team is 70% engineers
+- 1 in-house recruiter, no agencies
+
```

Comment:
it'd be fun / interesting to put the total applications we've ever received, if we have it

Reply:
i feel like it's worth making the point that we get a _ton_ of very automated applications (and why this happens and how it forces us to look so quickly at people's resumes)

Reply:
added a little more data here (I only have the last 12-ish months really as we switched from Workable to Ashby, but the number of applications per role is more impressive anyway as we don't hire that many people)